### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by PhantomG27249** [UltimateBoomer/smoothboot-fabric#18]
Will smooth boot becoming to 1.17?

**Comment by UltimateBoomer** [comment]
Yes, I'll definitely update the mod for 1.17. For the fabric version, the latest 1.16 version should already work with 1.17 (though I have not tested it yet myself),

**Comment by DMAgit** [comment]
Fabric release on Forge (1.16.5) works for 1.17.

**Comment by thefirethirteen** [comment]
The fabric 1.16.5 release also works for 1.17.1

**Comment by Obscure2020** [comment]
Really? Great! Thanks for the info.

**Comment by TheSnowyChickens** [comment]
Could you maybe update the Forge version to 1.18 ? :(

I don't like it to load a ~400 mods pack without smooth boot. :(

**Comment by jmdev91** [comment]
I assume it's just taking some time, dual-maintaining a given mod isn't easy especially when they go this far out of the designed use-case of the mod platforms, Happy to see the intent does seem to be to get to it when time allows, this mod literally makes or breaks Forge mod lists for me; without it I can't even load longer ones without my computer overheating.
